pkgs/development/interpreters/erlang/generic-builder.nix

```diff
@@ -81,7 +82,7 @@ in stdenv.mkDerivation ({
     ./otp_build autoconf
   '';
 
-  configureFlags = [ "--with-ssl=${openssl.dev}" ]
+  configureFlags = [ "--with-ssl=${opensslPackage.dev}" ]
```

Comment:
That would fail if the opensslPackage has no dev output. Is this fine? I don't really know.

Reply:
Thanks for you're review!

Since this is for people to override, it's a relatively advanced use in my opinion. It feels that if it people are using it, they will be able to diagnose why it failed.

I tried to find where that came from, I found the commit, but not information on why that is
https://github.com/NixOS/nixpkgs/commit/2a73de6e6c7899e34b8cd8f3a625e7c18d3ead33#diff-f5b7dbc04ba0ca072daeaf5979283b10ca66353bf749e1854fae65d80d4afc78

If you feel uncomfortable merging this change, I understand. Let me know and I'll try to get more opinions on it.

Reply:
@Mic92 when you have a moment, could we get your opinion on this if possible?